web/app.js

```diff
@@ -1178,7 +1183,10 @@ let PDFViewerApplication = {
       }
 
       if (pdfTitle) {
-        this.setTitle(pdfTitle + ' - ' + document.title);
+        this.setTitle(
+          `${pdfTitle} - ${contentDispositionFilename || document.title}`);
```

Comment:
`document.title` contains HTML viewer title. I think we just want to see following instead this patch:

    if (!pdfTitle && contentDispositionFilename) pdfTitle = contentDispositionFilename;

Reply:
Actually, it's more complicated than I tought, having `document.title` to hold application variable is totally confusing. The change is good.